rocky/rocky/bytes_client.py

```diff
@@ -128,6 +128,26 @@ def _save_raw(self, boefje_meta_id: uuid.UUID, raw: bytes, mime_types: Set[str]
 
         return response.json()[file_name]
 
+    def _save_raws(
+        self, boefje_meta_id: uuid.UUID, raws: list[tuple[uuid.UUID, bytes]], mime_types: Set[str] = frozenset()
+    ) -> dict[str, str]:
+        files = [
+            {
+                "name": str(normalizer_id),
+                "content": b64encode(raw).decode(),
+                "tags": list(mime_types) + [f"Declaration/{str(normalizer_id)}"],
```

Comment:
I think this can be removed now:
```suggestion
                "tags": list(mime_types),
```